### ISSUE TITLE
Enums not converted

### ISSUE THREAD

**Issue by bre7** [drkameleon/iswift#31]
Example:
```objectivec
typedef NS_ENUM (NSUInteger, TestEnum) {
    TestEnumCase1,
    TestEnumCase2,
    TestEnumCase3,
    TestEnumCase4,
    TestEnumCase5
};
```

**Output:**
```swift
enum TestEnum {
    case Case1, Case2, Case3, Case4, Case5
}
```

If objectivec compatibility is required (which should also be a setting imo...It changes a lot of things), then `@objc enum TestEnum: Int`

**Comment by drkameleon** [comment]
Well, this is a known (unimplemented) issue.

It's in my to-do list. Thanks for reminding me! :)

**Comment by drkameleon** [comment]
Fixed as of the upcoming 1.5. 

As for Objective-C compatibility, you sure have a point there, but I'll postpone that type of optimizations/customizations until the core is as stable as possible. :)

-----

**Example:**

```objective-c
typedef NS_ENUM (NSUInteger, TestEnum) {
    TestEnumCase1,
    TestEnumCase2,
    TestEnumCase3,
    TestEnumCase4,
    TestEnumCase5
};

typedef enum someEnum {
    kCircle,
    kRectangle,
    kOblateSpheroid
};
```

**Output:**

```swift
enum TestEnum {
	case TestEnumCase1, TestEnumCase2, TestEnumCase3, TestEnumCase4, TestEnumCase5
}

enum someEnum {
	case kCircle, kRectangle, kOblateSpheroid
}
```
